docker-compose.yml

```diff
@@ -6,12 +6,16 @@ services:
       context: .
       dockerfile: Dockerfile-mysterium-node
     container_name: mysterium-node
+    cap_add:
+      - NET_ADMIN
```

Comment:
I haven't had luck with `--cap-add=NET_ADMIN`. I needed to use `--privileged` for IP forwarding to work.